qa-core/src/internal-api/fixtures/datasources.ts

```diff
@@ -0,0 +1,57 @@
+// Add information about the data source to the fixtures file from 1password
+export const mongoDB = () => {
+  return {
+    datasource: {
+      name: "MongoDB",
+      source: "MONGODB",
+      type: "datasource",
+      config: {
+        connectionString: process.env.MONGODB_CONNECTION_STRING,
+        db: process.env.MONGODB_DB,
+      },
+    },
+
+    fetchSchema: false,
+  }
+}
+
+export const postgresSQL = () => {
+  return {
+    datasource: {
+      name: "PostgresSQL",
+      plus: true,
+      source: "POSTGRES",
+      type: "datasource",
+      config: {
+        database: process.env.POSTGRES_DB,
```

Comment:
Might be a good idea to add these new env vars to the root `environment.ts` file so we can see all env vars used by the suite at a glance